website/client/components/chat/chatMessages.vue

```diff
@@ -27,7 +27,7 @@
           span.action( @click='copyAsTodo(msg)')
             .svg-icon(v-html="icons.copy")
             | {{$t('copyAsTodo')}}
-          span.action(v-if='user.contributor.admin || (!msg.sent && user.flags.communityGuidelinesAccepted)', @click='report(msg)')
+          span.action(v-if='user.contributor.admin || (msg.uuid !== user._id && user.flags.communityGuidelinesAccepted)', @click='report(msg)')
```

Comment:
It needs to be possible for a user to report their own message. It's useful occasionally. I had to report one of mine a couple of days ago beccause I replied to a user who later was revealed to be underaged without permission, and it would have been appropriate for me to flag my own message then even if I wasn't a mod. There can also be other  reasons.